src/rt/sched/cown.h

```diff
@@ -11,6 +11,8 @@
 #include "priority.h"
 #include "schedulerthread.h"
 
+#include <algorithm>
```

Comment:
Why is this now needed?

Reply:
std::any_of in this file needs it and for the current platforms this is picked up from cpu.h, but for the new platforms we need an explicit include.